mappings/net/minecraft/world/ModifiableWorld.mapping

```diff
@@ -3,6 +3,32 @@ CLASS net/minecraft/class_1945 net/minecraft/world/ModifiableWorld
 	METHOD method_22352 breakBlock (Lnet/minecraft/class_2338;Z)Z
 		ARG 1 pos
 		ARG 2 drop
+	METHOD method_30092 setBlockState (Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;II)Z
+		COMMENT Updates the block state at a position, calling appropriate callbacks.
+		COMMENT
+		COMMENT <p>When called on the server, the new block state is stored and propagated to clients and listeners as dictated
+		COMMENT by the supplied flags. Note that calling this on the client will update the world locally, but may not see the
+		COMMENT change persisted across loads. It's recommended to check whether this world is client before
+		COMMENT interacting with the world in this way.</p>
+		COMMENT
+		COMMENT <p>Accepted values of the flags are a bitset combination of the following:</p>
+		COMMENT <ul>
+		COMMENT <li>0b0000001 // 1 - PROPAGATE_CHANGE - Propagates a change event to surrounding blocks.</li>
+		COMMENT <li>0b0000010 // 2 - NOTIFY_LISTENERS - Notifies listeners and clients who need to react when the block changes</li>
+		COMMENT <li>0b0000100 // 4 - NO_REDRAW - Used in conjunction with NOTIFY_LISTENERS to suppress the render pass on clients.</li>
+		COMMENT <li>0b0001000 // 8 - REDRAW_ON_MAIN_THREAD - Forces a synchronous redraw on clients.</li>
+		COMMENT <li>0b0010000 // 16 - FORCE_STATE - Bypass virtual blockstate changes and forces the passed state to be stored as-is.</li>
```

Comment:
‘Blockstate’ should be two words here, but this is an issue on [the other `setBlockState` method](https://github.com/FabricMC/yarn/blob/c72ab7649ee46d512df5ce90a11cea7b6cec28f6/mappings/net/minecraft/world/ModifiableWorld.mapping#L29) so it should be fixed in a separate issue.

Reply:
I've opened #1497 to fix this issue.